### ISSUE TITLE
Combined ex commands deviate from Vim behavior

### ISSUE THREAD

**Issue by mcp292** [emacs-evil/evil#1719]
# MWE
```txt
1
2
3
4
5
```

With cursor on the line with `5`:
1. Yank 3 lines up and put below: `:-3y | put`.

Should get:
```txt
1
2
3
4
5
2
```

# Problem
Spacemacs is giving: `Invalid count` in the minibuffer.

**Comment by tomdl89** [comment]
Yes, bar is not implemented in evil's ex. It requires quite a lot of work, because we need to manually tag every ex command as either supporting it or not (because some commands see it as an argument and others don't). PR welcome, although I do expect to take a look one day.

**Comment by mcp292** [comment]
Does the list in [`:help :bar`](https://vimdoc.sourceforge.net/htmldoc/cmdline.html#:bar) not cover it?

**Comment by tomdl89** [comment]
Probably. My point isn't that someone has to work out what that list is. My point is that someone has to go through every equivalent in evil and tag it, then add logic to handle the tags. Doable, but not a trivial amount of work. Not time that I have found recently.

**Comment by mcp292** [comment]
I am not a lisp programmer, but I can volunteer some time if you get me started. I can at least cover the contents of that list, and users and maintainers can add to it as they're found.

**Comment by axelf4** [comment]
Duplicate of #584
